### ISSUE TITLE
Installation fails if gpg has a "trusted-key" set

### ISSUE THREAD

**Issue by lp0** [keybase/node-installer#69]
With a "trusted-key" set in gpg.conf, the installation will fail because that key won't exist in the new empty keyring.

<pre>
debug: |||||| Mutate GPG args; new args: --no-default-keyring --keyring /tmp/keybase_install_7rp50JGsSmhhrVsIwQny5A/QoaZnEWIKXSjUIB2lQaA/pubring.gpg --secret-keyring /tmp/keybase_install_7rp50JGsSmhhrVsIwQny5A/QoaZnEWIKXSjUIB2lQaA/secring.gpg --trustdb-name /tmp/keybase_install_7rp50JGsSmhhrVsIwQny5A/QoaZnEWIKXSjUIB2lQaA/trustdb.gpg --import
info: Using keyset version v1
debug: ------ Installed: -> GpgError: exit code 2
debug: ++++++ cleanup /tmp/keybase_install_7rp50JGsSmhhrVsIwQny5A
info: cleaning up tmpdir /tmp/keybase_install_7rp50JGsSmhhrVsIwQny5A
debug: |||||| Unlink /tmp/keybase_install_7rp50JGsSmhhrVsIwQny5A/QoaZnEWIKXSjUIB2lQaA
error: In cleanup: Error: EISDIR, unlink '/tmp/keybase_install_7rp50JGsSmhhrVsIwQny5A/QoaZnEWIKXSjUIB2lQaA'
error: exit code 2
warn: gpg: key 049F5C6C11111111: no public key for trusted key - skipped
warn: gpg: key 049F5C6C11111111 marked as ultimately trusted
warn: gpg: key 47484E50656D16C7: public key "Keybase.io Code Signing (v1) <code@keybase.io>" imported
warn: gpg: Total number processed: 1
warn: gpg:               imported: 1  (RSA: 1)
warn: gpg: public key of ultimately trusted key 049F5C6C11111111 not found <-------
warn: gpg: 3 marginal(s) needed, 1 complete(s) needed, PGP trust model
warn: gpg: depth: 0  valid:   1  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 1u
</pre>

**Comment by lp0** [comment]
This also causes most other functions to fail too because it creates a temporary keyring.

**Comment by mjpieters** [comment]
I see this too; temporarily commenting out the `trusted-key` entry allowed me to run the installer successfully.

**Comment by mjpieters** [comment]
Of course, you have to continue commenting out the entry if you want to do anything with `keybase` that involves signing.

**Comment by jaredallard** [comment]
I ran across this issue, however just running gpg fixed it.
